apps/hello/templates/services/app/configs/test.json.j2

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Have you tried to mount this configMap in the hello app DC?

Reply:
I didn't try to mount the configMap because if the object is deployed on kubernetes without any error, I assume that the configMap is mountable.

I checked its content manually and I added the `test.json.j2` file to the project to ensure that the playbook doesn't fail on the CI.

To have a complete test suite, I can test the validity of the generated json file by mounting it on a container with `jq` utility installed, but do you think it's worth it ?

Reply:
Nope. Go on!